src/s3-copy.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env python3
```

Comment:
This is not needed and can cause problems if the Python executable is not in that path (i.e., if you used the UFS-RNR python stack of RDHPCS Hera and Orion).